code/game/objects/items/stacks/stack.dm

```diff
@@ -657,7 +657,8 @@
 
 /obj/item/stack/hitby(atom/movable/hitting, skipcatch, hitpush, blocked, datum/thrownthing/throwingdatum)
 	if(can_merge(hitting, inhand = TRUE))
-		merge(hitting)
+		if(merge(hitting))
+			playsound(src, drop_sound, PICKUP_SOUND_VOLUME, sound_vary, ignore_walls = FALSE) // same settings as pickup
```

Comment:
```suggestion
```
I couldn't apply a single suggestion because it detected a deleted line in the middle :T